<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Составление вариаций диагноза</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="gb_main">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Основное заболевание:</string>
      </property>
      <widget class="QPushButton" name="pb_addMain">
       <property name="geometry">
        <rect>
         <x>385</x>
         <y>31</y>
         <width>51</width>
         <height>101</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>101</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gb_back">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Фоновое заболевание:</string>
      </property>
      <widget class="QPushButton" name="pb_addBack">
       <property name="geometry">
        <rect>
         <x>385</x>
         <y>31</y>
         <width>51</width>
         <height>101</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>101</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gb_comp">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Осложнения основного заболевания:</string>
      </property>
      <widget class="QPushButton" name="pb_addComp">
       <property name="geometry">
        <rect>
         <x>385</x>
         <y>31</y>
         <width>51</width>
         <height>101</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>101</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gb_acc">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Сопутствующие заболевания:</string>
      </property>
      <widget class="QPushButton" name="pb_addAcc">
       <property name="geometry">
        <rect>
         <x>385</x>
         <y>31</y>
         <width>51</width>
         <height>101</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>101</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QPushButton" name="pb_saveDiagnosis">
        <property name="minimumSize">
         <size>
          <width>261</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Сохранить диагноз</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pb_further">
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Далее</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lb_diagnosisCount">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Сохранено вариаций диагноза: 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
